.github/workflows/wiki.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish Wiki
+
+# Trigger this action only if there are changes pushed to the wiki/** directory under the main branch
+on:
+  push:
+    paths:
+      - docs/** # This includes all sub folders
+    branches:
+      - main # This can be changed to any branch of your preference
+
+env:
+  USER_TOKEN: <USER_TOKEN_ENV_VARIABLE> # This is the repository secret
+  USER_NAME: <USER_NAME> # Enter the username of your (bot) account
+  USER_EMAIL: <USER_EMAIL> # Enter the e-mail of your (bot) account
```

Comment:
Do these need to be filled out with anything?

Reply:
Yes they do, for the CI to run, there needs to be the environment variable for the user token secret, kind of like `secrets.GITHUB_TOKEN` that you have in `release.yml` workflow.
for the name and email, those are for the commits, so we can see in the wiki where the updates come from. It usually shows as "revisions".

Reply:
Sorry for getting to this so late. So I need to change those myself? Or can you do it in this PR?

Reply:
I don't really know which secret to use to access the wiki,  so what I did for now is I changed them to these 

```
env:
  USER_TOKEN: ${{ secrets.WIKI_TOKEN }}
  USER_NAME: github-actions[bot]
  USER_EMAIL: github-actions[bot]@users.noreply.github.com
```

I cannot really use the same `GITHUB_TOKEN` you use in the other workflows because the wiki is technically a different repository, so the secret for this repository won't work for it as far as I know, so I just put a `WIKI_TOKEN`. 

Now what you need to do for this to work (hopefully if I didn't miss anything): 

1. Go to GitHub Settings → Developer Settings → Personal Access Tokens
2. Generate a new token with wiki permissions
3. Add it to your repository's secrets as `WIKI_TOKEN`

Reply:
Gotcha thanks. Sorry again for the delay, I've been really busy in my personal life but I'll try to get to this soon

Reply:
Of course, the advantage here is we don't have a deadline, so take your time ! No pressure. The last thing we want is for the maintainer to have a burnout ;)